### PR TITLE
Fix adding option for generating json schema for given ballerina service file

### LINKED ISSUE #93: Add option for generating json schema for given ballerina service file
**Description:**
Generate JSON schema for given ballerina file instead of generating its YAML file.
When user needs json schema from given ballerina file, can add this option `--json` to command
example
`bal openapi -i <ballerina file> --json`

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by lnash94**
> Generate JSON schema for given ballerina file instead of generating its YAML file.
When the user needs JSON schema from given ballerina file, can add this `--json` option to ballerina to openapi command. Please Note: Generating json file is available for the services already supported to ballerina-to-openapi command.

example
`bal openapi -i <ballerina file> --json`

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: openapi-cli/src/main/java/org/ballerinalang/openapi/cmd/OpenApiCmd.java [hunk 1]
```diff
@@ -156,8 +159,14 @@ private void ballerinaToOpenApi(String fileName) throws IOException {
 //        Path resourcePath = getRelativePath(new File(balFilePath.toString()), this.targetOutputPath.toString());
         //ballerina openapi -i service.bal --serviceName serviceName --module exampleModul -o ./
         // Check service name it is mandatory
+        boolean isJson;
+        if (generatedFileType) {
```
Comment (hemikak) [inline openapi-cli/src/main/java/org/ballerinalang/openapi/cmd/OpenApiCmd.java:163]: Can this be null ?

Comment (hemikak) [inline openapi-cli/src/main/java/org/ballerinalang/openapi/cmd/OpenApiCmd.java:163]: You can directly assign isJson to generatedFileTypes right ?

**Comment by hemikak** [comment]
Can we add some test cases as well ?

File: openapi-cli/src/main/java/org/ballerinalang/openapi/cmd/OpenApiCmd.java (hunk 1 above)
Comment (lnash94) [reply openapi-cli/src/main/java/org/ballerinalang/openapi/cmd/OpenApiCmd.java:163]: yeah, I will directly assign this value.

**Review by lnash94 — COMMENTED** [review]
> Can we add some test cases as well ?

Will add tests for this

**Review by hemikak — APPROVED** [review]

**Pull request merged**